.changeset/wide-ghosts-own.md

```diff
@@ -0,0 +1,12 @@
+---
+"@pnpm/core": minor
+"@pnpm/config": minor
+"@pnpm/plugin-commands-installation": patch
+"@pnpm/plugin-commands-patching": patch
```

Comment:
Heads up that we usually add `pnpm: minor` as well for new features to indicate what the next pnpm binary version should be.

```suggestion
"@pnpm/plugin-commands-patching": patch
pnpm: minor
```